cloud/shared/bin/deploy.py

```diff
@@ -5,7 +5,7 @@
 from cloud.shared.bin.lib.config_loader import ConfigLoader
 
 
-def run(config: ConfigLoader):
+def run(config: ConfigLoader, params=None):
```

Comment:
Given that run() now always called with 2 arguments and the second argument is always List[str], make it `run(config: ConfigLoader, params: List[str])`?